version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "3.8.1-SNAPSHOT"
+ThisBuild / version := "3.9.0-SNAPSHOT"
```

Comment:
you should not bump version manually, please do it as part of `sbt release`

Reply:
It will offer "3.9.0" and I'll just accept it